### ISSUE TITLE
How was your SUN RGB-D AP@15 calculated exectly？

### ISSUE THREAD

**Issue by YangYang-DLUT** [ByteDance-Seed/Seed1.5-VL#31]
![Image](https://github.com/user-attachments/assets/31d99ad4-a29e-4134-aaac-315d9891da70)
More details needed here.

**Comment by yanwei-li** [comment]
Thanks for your interest in our work! We utilize the classic 3D AP claculation pipeline, similar to AP15 using in [Omni3D](https://github.com/facebookresearch/omni3d/blob/main/cubercnn/evaluation/omni3d_evaluation.py). Thanks!

**Comment by zjh-tsinghua** [comment]
Hello, I would like to ask about the evaluation results of AP@15 on the SUNRGB-D dataset. Could you please clarify which evaluation sets were used for this? Additionally, which other objects were considered in the evaluation?

Thank you!

**Comment by yanwei-li** [comment]
Hi, for SUNRGB-D dataset, we use the [Total3D](https://github.com/GAP-LAB-CUHK-SZ/Total3DUnderstanding) version with 33 categories for evaluation. We also consider other commen indoor object evaluation, like ARKitScenes_test used in [Omni3D](https://github.com/facebookresearch/omni3d/blob/main/RESULTS.md).

> Hello, I would like to ask about the evaluation results of AP@15 on the SUNRGB-D dataset. Could you please clarify which evaluation sets were used for this? Additionally, which other objects were considered in the evaluation?
> 
> Thank you!

**Comment by zjh-tsinghua** [comment]
Hello,
I have a couple of clarification questions regarding the evaluation protocol:

**1. Evaluation Set Image IDs**
Could you please confirm which images are used for evaluation? Are you using the validation set from SUNRGB-D? If so, does it correspond to image IDs 1–5050?

**2. Number of Classes: 33 vs. 37**
I noticed that you seems to be using 33 classes, while the original SUNRGB-D segmentation annotations and  [Total3D](https://github.com/GAP-LAB-CUHK-SZ/Total3DUnderstanding) use 37 classes. Could you clarify which 33 categories are being used for evaluation? A class list would be greatly appreciated.

Thank you again for your efforts, and I look forward to your response!

**Comment by Yangr116** [comment]
Hi @zjh-tsinghua , please refer to Table-10 of total-3d paper(https://arxiv.org/pdf/2002.12212)

<img width="703" height="342" alt="Image" src="https://github.com/user-attachments/assets/82d93af6-9164-469c-83c0-aa3854f1246f" />

**Comment by zjh-tsinghua** [comment]
Could you please share the performance of SeedVL-1.5 on the 33 categories if available?

It would be very helpful for comparison. Thanks in advance!

Meanwhile, I want to check if you used all 10325 images to evaluate or only use 1-5050 images , which are in sunrgbd_val ?

**Comment by yanghtr** [comment]
@yanwei-li @Yangr116 

Thanks for the great work. I have a couple of questions regarding the SUNRGBD evaluation:

- From the 3D Cookbook, I noticed that the model outputs results in the format
<3dbbox>x_1 y_1 z_1 x_size_1 y_size_1 z_size_1 pitch_1 yaw_1 roll_1</3dbbox><3dbbox>x_2 y_2 z_2 x_size_2 y_size_2 z_size_2 pitch_2 yaw_2 roll_2</3dbbox>...
without object categories. During the SUNRGBD evaluation, do you prompt the model 33 separate times, each time manually specifying a single object category? Could you also share what question prompt you used for these evaluations?

- Could you clarify whether the evaluation was conducted on the sunrgbd_val split (5050 images)? Additionally, did you include the SUNRGBD training set during model training?
